feature-configs/5g-ref-nw/clusters/otwaon1234rd/remote-sites/otwaon23456rw/nw-config/sriov/sriov-networks.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-fec-fh
+  namespace: openshift-sriov-network-operator
+spec:
+  vlan: 142
+  resourceName: du_fec_fh_otwaon23456rw
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-fec-mh-c
+  namespace: openshift-sriov-network-operator
+spec:
+  vlan: 100
+  resourceName: du_fec_mh_otwaon23456rw
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-fec-mh-u
+  namespace: openshift-sriov-network-operator
+spec:
+  vlan: 101
+  resourceName: du_fec_mh_otwaon23456rw
```

Comment:
We need to look at this part.  I think we need a new namespace since the N3000 does not use the generic SRIOV operator for configuration.  This is an example that I have seen:

apiVersion: sriovfec.intel.com/v1
kind: SriovFecClusterConfig
metadata:
  name: config
  namespace: n3000-operators-resources
spec:
  nodes:
    - nodeName: node1
      physicalFunctions:
        - pciAddress: 0000.1d.00.0
          pfDriver: pci-pf-stub
          vfDriver: vfio-pci
          vfAmount: 2
          bbDevConfig:
            n3000:
              # Network Type: either "FPGA_5GNR" or "FPGA_LTE"
              networkType: "FPGA_5GNR"
              # Programming mode: 0 = VF Programming, 1 = PF Programming
              pfMode: false
              flrTimeout: 610
              downlink:
                bandwidth: 3
                loadBalance: 128
                queues:
                  vf0: 16
                  vf1: 16
                  vf2: 0
                  vf3: 0
                  vf4: 0
                  vf5: 0
                  vf6: 0
                  vf7: 0
              uplink:
                bandwidth: 3
                loadBalance: 128
                queues:
                  vf0: 16
                  vf1: 16
                  vf2: 0
                  vf3: 0
                  vf4: 0
                  vf5: 0
                  vf6: 0
                  vf7: 0

Reply:
I think this is a naming problem and confusion that will be resolved after I fix all the comments made by William. 
Here 'fec' is Far Edge Cloud, related to the profile of a single-socket server.
In Intel's sriov fec objects, 'fec' means forward error correction.
So yes, need to fix, but not to n3000 config, just the sriovResourceName must become more clear